lms/djangoapps/courseware/tests/test_views.py

```diff
@@ -302,8 +301,8 @@ def setUp(self):
             )
 
         self.course_key = self.course.id
-        self.password = '123456'
-        self.user = UserFactory(username='dummy', password=self.password, email='test@mit.edu')
+        # Set profile country to Åland Islands to check special characters does not raise error
+        self.user = UserFactory(username='dummy', profile__country='AX')
```

Comment:
Remove the username kwarg.

Reply:
@clintonb It being used in `ddt.data` of a test case https://github.com/edx/edx-platform/blob/e4d5a2a3e9972f3870f2f64b5df4082e5f1888a3/lms/djangoapps/courseware/tests/test_views.py#L913

Reply:
I fail to see why that matters. That's a completely different test case. If that test needs a user with the specified username, that test should create the user. Test order is not static, and tests should always be independent of one another.